Tribler/community/tunnel/tunnel_community.py

```diff
@@ -1161,7 +1162,10 @@ def exit_data(self, circuit_id, sock_addr, destination, data):
                     str(sock_addr), str(self.exit_sockets[circuit_id].sock_addr))
                 self.exit_sockets[circuit_id].enable()
             try:
-                self.exit_sockets[circuit_id].sendto(data, destination)
+                if destination[0] != 'fake.net':
```

Comment:
Saw that the error was already catched but there was a print_exc which resulted in the traceback. It's replaced by a logged error with destination included.